docker/buildkite/docker-compose.yml

```diff
@@ -73,6 +78,7 @@ services:
       context: ../../
       dockerfile: ./docker/buildkite/Dockerfile
     environment:
+      - "CASSANDRA_HOST=cassandra"
```

Comment:
huh.  was this needed?  maybe due to install-schema?
most things should be using CASSANDRA_SEEDS (which is an awful name, _HOST is much better)

Reply:
yes